### ISSUE TITLE
Making frontend responsive

### ISSUE THREAD

**Issue by Shashank024** [JINDAL-JJ/UniVendor#5]
Set up media queries to make the frontend look better.

Currently, products shrink directly from 3 to 1 on shortening the width of the window. Enhance it to first shrink to two products in one row than one.

**Comment by Thiyageshkanna** [comment]
Hey !! I would like to work on this

**Comment by JINDAL-JJ** [comment]
Yeah go ahead and submit the PR, but ensure that you have to make it in 3-4 days as the issue is easy!

**Comment by Shashank024** [comment]
@Thiyageshkanna You are working on this issue or not? If you are facing any problem, let us know on our discord server and make the PR soon asap as we give your more than expected time to make the PR.
